src/app/Scenes/Activity/components/NotificationArtworkList.tsx

```diff
@@ -3,17 +3,26 @@ import { Flex, Screen } from "@artsy/palette-mobile"
 import { NotificationArtworkList_artworksConnection$key } from "__generated__/NotificationArtworkList_artworksConnection.graphql"
 import { PriceOfferMessage } from "app/Components/ArtworkGrids/ArtworkGridItem"
 import { MasonryInfiniteScrollArtworkGrid } from "app/Components/ArtworkGrids/MasonryInfiniteScrollArtworkGrid"
+import { CommercialButtonsQueryRenderer } from "app/Scenes/Activity/components/NotificationCommercialButtons"
 import { extractNodes } from "app/utils/extractNodes"
 import { FC } from "react"
 import { useFragment, graphql } from "react-relay"
 
+export interface PartnerOffer {
+  endAt: string
+  isAvailable: boolean
+  targetHref: string
+}
```

Comment:
Personally, I find it easier to read when there are empty lines between larger blocks like these. 🤷

```suggestion
}

```